chromium_src/third_party/blink/renderer/DEPS

```diff
@@ -1,4 +1,5 @@
 include_rules = [
+  "+../gen/third_party/blink/renderer/bindings/modules/v8",
```

Comment:
I think we should automate `+..gen/` DEPS rule as we do for `src/` now.
Please add this:
```
      # Add `+../gen/...` rule.
      root_src_relative_dir = '../gen/' + posixpath.relpath(
          root_src_dir_path_norm, NormalizePath(self.base_directory))
      directory_rules.AddRule('+' + root_src_relative_dir,
                              root_src_relative_dir,
                              'Gen rule for ' + root_src_relative_dir)
```
[right after this line](https://github.com/brave/brave-core/blob/587a51ad4f2508e650d3ddef8008404da3c6512b/chromium_src/python_modules/buildtools/checkdeps/brave_builddeps.py#L32).

And cleanup these two files manually:
`brave/chromium_src/chrome/grit/DEPS`
`brave/chromium_src/third_party/blink/renderer/DEPS`